src/style.css

```diff
@@ -1,3 +1,68 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 50px;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+}
```

Comment:
If there is no requirement to fix the size of containers, it's better not to assign explicit widths and heights to elements like `.header__logo` unless necessary. Let the content size dictate it to avoid overflow or accidental scroll bars.